.gitpod.yml

```diff
@@ -2,8 +2,6 @@
 # Please adjust to your needs (see https://www.gitpod.io/docs/config-gitpod-file)
 # and commit this file to your remote git repository to share the goodness with others.
 
-tasks:
-  - init: go get && go build ./... && go test ./... && make
-    command: go run
+image: ghcr.io/notional-labs/images/gno
```

Comment:
Merging 178, allows us to change this import path to:

github.com/gnolang/gno <- with compiled binaries

github.com/gnolang/devenv <- without binaries